src/ts/order.ts

```diff
@@ -69,16 +69,15 @@ export type Timestamp = number | Date;
 /**
  * Order kind.
  */
-export const enum OrderKind {
-  /**
-   * A sell order.
-   */
-  SELL,
-  /**
-   * A buy order.
-   */
-  BUY,
-}
+export type OrderKind = "sell" | "buy";
```

Comment:
Why not
```ts
enum OrderKind {
  SELL = 'sell',
  BUY = 'buy'
}
```
You don't have to switch to `type`

Reply:
:open_mouth: Last time I tried this you had to
```ts
enum OrderKind {
  SELL = 'sell' as any,
  BUY = 'buy' as any,
}
```

I guess I'm just getting old :stuck_out_tongue:

Reply:
No need for `const OrderKinds` then
And no need for half the changes in this OR then...

Reply:
I thought the "not using enums" was for the https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats

Reply:
`const enums` are icnompatible with babel-ts without workarounds
not normal _object_ enums
[compare](https://www.typescriptlang.org/play?target=99#code/PTAEHsCMCsFMGMAuoCWA7UApAygKFmgK4C2oA8gE4AmsFA0ulaAN66ijYCiAMt6ALygA5AGdYAG3FCANG1AAhAKoBNAcMiEAnjNwBfXPHBoR4cbAB048AHMAFJRr1G5pcoCUuA0ZHICJctS0DGhUAMIscly8aqISUrLsrjEa2rL6IKho4uiwTACGyIbEAA4oZgC0iCjEsF7GphZWdg5BjKEuKm5AA)

Reply:
Then, yep, enum is the natural thing to do there

Reply:
Agreed.